src/simulation/dynamics/FuelTank/fuelTank.h

```diff
@@ -262,8 +263,8 @@ class FuelTank :
 public:
     std::string nameOfMassState{};                       //!< -- name of mass state
     std::vector<FuelSlosh *> fuelSloshParticles;        //!< -- vector of fuel slosh particles
-    std::vector<DynamicEffector *> dynEffectors;        //!< -- Vector of dynamic effectors for thrusters
-    std::vector<StateEffector *> stateEffectors;        //!< -- Vector of state effectors for thrusters
+    std::vector<ThrusterDynamicEffector *> dynEffectors;        //!< -- Vector of dynamic effectors for thrusters
```

Comment:
Since you made it explicit to be ThrusterDynamicEffectors and ThrusterStateEffectors, doesn't it make sense to change the variable names too?

Reply:
I agree, this has been changed now